Fabric.Identity.API/scripts/tests/identity-cli-tests.ps1

```diff
@@ -0,0 +1,59 @@
+# Force re-import to pick up latest changes
+Import-Module ..\identity-cli.psm1 -Force
+
+describe 'Get-AccessToken' {
+    Context 'Unit Tests' {
+        Context 'Valid Request' {
+            It 'Should return an access token when valid response' {                
+                $mockUrl = New-MockObject -Type Uri
+                Mock -ModuleName identity-cli -CommandName Invoke-RestMethod { return @{
+                    access_token = "return token"
```

Comment:
Should we mock the full response that the server would send back, to make sure that our code is parsing and returning the right value?